x-pack/plugins/security_solution/public/timelines/components/timeline/eql_tab_content/index.tsx

```diff
@@ -298,6 +299,8 @@ export const EqlTabContentComponent: React.FC<Props> = ({
                   itemsCount: totalCount,
                   itemsPerPage,
                 })}
+                leadingControlColumns={[defaultControlColumn]}
```

Comment:
Consider creating constants (outside of the component's closure) for the arrays passed to the `leadingControlColumns` and `trailingControlColumns` props below:

```
                leadingControlColumns={[defaultControlColumn]}
                trailingControlColumns={[]}
```

e.g.

```
const leadingControlColumns = [defaultControlColumn];
const trailingControlColumns = [];

// ...

                leadingControlColumns={leadingControlColumns}
                trailingControlColumns={trailingControlColumns}
```

to ensure the same array references are passed as props.